docs/user/getting-startedv2.rst

```diff
@@ -0,0 +1,116 @@
+.. _getting_startedv2:
+
+Getting Started V2
+==================
+This section takes you through a the entire process of conducting a co-simulation with Maestro2.
+
+| This guide is based on the command line interface (CLI) of Maestro2 and consists of two parts: 
+| The first part concerns writing a MaBL Specification by hand and executing it.
+| The second part concerns using the specification generation and expansion capabilities of Maestro.
+
+Note that Maestro2 also features a web API presented in :ref:`web-api`.
+
+Setup
+-----
+The first step is to install the dependencies along with the tools and resources that are to be used
+
+- Download and install Java 11
+- Download the :download:`example resources part1 <images/wt_example/watertank-example-part1.zip>` and unzip into a directory.
```

Comment:
maybe just list the real dependencies + tool and save the example zip's to their section

Reply:
Done